app/kubernetes-ingress-controller/2.0.x/guides/using-kongplugin-resource.md

```diff
@@ -291,14 +283,14 @@ demo:  injected-by-kong
 Server: kong/1.2.1
 ```
 
-You can also see how the `demo` header was injected as the request also
-matched one of the rules defined in the `demo` `Ingress` resource.
+You can also see how the `demo` header was injected only for `/foo`,
+as the request also matched one of the rules defined in the `demo` `Ingress`
+resource but `/baz` does not match.
```

Comment:
```suggestion
You can also see how the `demo` header was injected only for `/foo`,
as the request matched one of the rules defined in the `Ingress`
resource, but not for `/baz` because that request does not match.
```